backend/src/packages/map/map.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { ApiPath } from '~/libs/enums/enums.js';
+import {
+  type ApiHandlerOptions,
+  type ApiHandlerResponse,
+  Controller,
+} from '~/libs/packages/controller/controller.js';
+import { HttpCode } from '~/libs/packages/http/http.js';
+import { type ILogger } from '~/libs/packages/logger/logger.js';
+
+import { AuthStrategy } from '../auth/auth.js';
+import { type UserEntityObjectWithGroupT } from '../users/users.js';
+import { MapApiPath } from './libs/enums/enums.js';
+import { type CalculatePriceRequest } from './libs/types/types.js';
+import { type MapService } from './map.service.js';
+
+class MapController extends Controller {
+  private mapService: MapService;
+
+  public constructor({
+    logger,
+    mapService,
+  }: {
+    logger: ILogger;
+    mapService: MapService;
+  }) {
+    super(logger, ApiPath.MAP);
+
+    this.mapService = mapService;
+
+    this.addRoute({
+      path: MapApiPath.ROOT,
```

Comment:
the route for calculating the price should be in order controller
/orders/calculate-price